src/zope/testrunner/options.py

```diff
@@ -243,6 +243,16 @@
 When there is a doctest failure, show it as a context diff.
 """)
 
+reporting.add_option(
+    '--ignore_new_thread',
+    action="append",
+    default=[],
+    dest='ignore_new_threads',
+    help="""\
+If a thread with this name is left behind, don't report this at the end.
```

Comment:
What if a user desires to ignore all threads?  Perhaps it would be a good idea to support wildcards?